### ISSUE TITLE
check_subrc should not be raised if sy-tabix is checked.

### ISSUE THREAD

**Issue by pokrakam** [abaplint/abaplint#3418]
We can check sy-tabix for success. Playground:
```abap
REPORT zfoobar.

START-OF-SELECTION.

  DATA itab TYPE STANDARD TABLE OF c WITH EMPTY KEY.

  itab = VALUE #( ( 'a' ) ( 'b' ) ( 'c' ) ).
  READ TABLE itab WITH KEY table_line = 'b' TRANSPORTING NO FIELDS.
  IF sy-tabix <> 0.
    WRITE / 'Found'.
  ENDIF.
```
